docs/virtual_machines/templates.md

```diff
@@ -177,9 +224,8 @@ vice versa).
 
 The templates provided by the kubevirt project provide a set of
 conventions and annotations that augment the basic feature of the
-[openshift
-templates](https://docs.okd.io/latest/dev_guide/templates.html). You can
-customize your kubevirt-provided templates editing these annotations, or
+[openshift templates](https://docs.okd.io/latest/openshift_images/using-templates.html). 
```

Comment:
The OKD docs don't talk about virtual machine templates.  We should use a Red Hat OpenShift doc link right here.